model/gen-ai/registry.yaml

```diff
@@ -368,3 +368,33 @@ groups:
         type: string
         brief: A fingerprint to track any eventual change in the Generative AI environment.
         examples: ["fp_44709d6fcb"]
+  - id: registry.gen_ai.watsonxai
+    type: attribute_group
+    display_name: Watsonx AI Attributes
+    brief: >
+      Thie group defines attributes for Watsonx AI.
+    attributes:
+      - id: gen_ai.watsonxai.request.decoding_method
+        stability: development
+        type:
+          members:
+            - id: sample
+              value: "sample"
+              brief: 'Offers more variability in how tokens are selected'
```

Comment:
can we rephrase it to me more descriptive or link to external docs? more variability than what?